.github/workflows/build-and-deploy-master-certbot.yml

```diff
@@ -0,0 +1,101 @@
+name: "Build & Deploy Certbot"
+
+on:
+  push:
+    branches:
+      - lisaa-workflow-build-certbot
+      - master
+
+env:
+  ARTIFACT_NAME: koski-luovutuspalvelu-certbot
```

Comment:
Nääkin vois olla secreteissä, niin voi muokata ilman muutoksia deploy-skriptiin

Reply:
Ihan hyvä pointti 🤔 Ehkä se on hyvä olla kovakoodamatta näitä tälleen